controllers/authController.js

```diff
@@ -69,3 +69,34 @@ exports.register = async (req, res) => {
     })
   }
 }
+
+exports.resetPass = async (req, res) => {
+  try {
+    const userToSearchFor = { email: req.body.email, password: req.body.oldPass }
+    const newUserToSearchFor = { email: req.body.email, password: req.body.newPass }
+    const user = await req.db.User.findOne(userToSearchFor)
+    if (user === null) {
+      throw new Error('Wrong email or password')
+    }
+    const userChanged = await req.db.User.updateOne(userToSearchFor, { $set: { password: req.body.newPass } })
+    if (userChanged.nModified !== 1) {
```

Comment:
I think if mongo could not update the entry will throw an error so we shouldn't check that.